pagefaultgames/pokerogue
# [BETA] Read title from env files to allow for environment base changes

## What are the changes?


Make `<title>` content be read from `VITE_TITLE`

## Why am I doing these changes?




This allows to vary the title based on the environment

## What did change?



content of `<title>` is now `%VITE_TITLE%` which reads from `.env.<environment>#VITE_TITLE`

### Screenshots/Videos



<img width="198" alt="image" src="https://github.com/pagefaultgames/pokerogue/assets/50131232/a99dce41-6bef-4c52-819a-ca6075ebba84">
<img width="109" alt="image" src="https://github.com/pagefaultgames/pokerogue/assets/50131232/73d10cb1-bda8-45c5-95b9-4ae017528076">
<img width="173" alt="image" src="https://github.com/pagefaultgames/pokerogue/assets/50131232/6c24fbeb-205e-4b4b-b979-d55970c7049e">


## How to test the changes?





1. Launch locally
2. Check title
3. Change content of `.env` or `.env.development` or whatever your active is
4. reload
5. check title again -> Should be the content you set

## Checklist
